client/src/views/admin/AdminList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <b-container>
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Name</th>
+          <th scope="col">Role</th>
+          <th scope="col">Email</th>
```

Comment:
place admin list file inside `/components/Admin/AdminList.vue`

all components like dashboard, userlist, etc of `Admin` view must be placed in components/admin
update route as well

Reply:
Updated